### DISCUSSION TITLE
Notify plugin: add onClick event to configuration options

### DISCUSSION THREAD

**Discussion by elste** [quasarframework/quasar#16807] [Ideas / Proposals]
Hi, 
I would like to work on the **Notify plugin**.
My goal is to **close notifications by clicking on them**, without using actions.
I couldn't find anything about it in the documentation.
So I want to **add an onClick event to configuration options**.
I initially considered using a boolean property (e.g. closeOnClick), but I think onClick event can provide a higher level of abstraction and more ways of using it.

Before I start working on it, I would like to know your opinions and any advice.

Thank you

**Comment by gioboa** [comment]
Wow 🎉 this should be useful for my project 🤩
